package.json

```diff
@@ -81,6 +81,7 @@
     "zxcvbn": "^4.4.2"
   },
   "scripts": {
+    "postinstall": "npm run copy-fonts",
```

Comment:
Is this supposed to aid also in things like this: (where it seems like the OS/browser fonts are taking over our custom fonts)?
![image](https://user-images.githubusercontent.com/16762675/129054787-6eba36a6-7718-4d9a-8a8b-3a1f62a8a0d9.png)
I've been having this issue for a while and I'm not sure why :P the fonts throughout KPI look a bit off/wider than they're supposed to

Reply:
Nevermind! The above is a result of different Roboto fonts coming from my OS. On Ubuntu it produces the above but on Arch it's using the correct font.